src/code/sleep_forever.c

```diff
@@ -1,6 +1,6 @@
 #include "libc64/sleep.h"
 
-void func_80183070(void) {
+void Sleep_Forever(void) {
```

Comment:
So taking another look at OoT, they have an additional log message that uses the word freeze which I'm wondering if should actually use in the function name (`func_800D31A0` is the function in OoT).

Reply:
For reference, OoT (https://github.com/zeldaret/oot/pull/2315) looks to rename the file to `sys_freeze` (no word on the function yet, though I think just `Freeze` or `Freeze_Forever` works) which I agree with.

Reply:
I disagree with OoT's decision to call it `sys_freeze`. The word freeze in the context most appropriate is a system hang, or an unintended fault of software/hardware causing unresponsiveness, and not a deliberate action. Additionally, calling `Sleep_Forever` does not necessarily result in an unresponsive system.

The "freeze!" comment in OoT more seems like a joke than a hint at the actual source name.